pkg/apis/extensions/v1beta1/types.go

```diff
@@ -23,22 +23,22 @@ import (
 	"k8s.io/kubernetes/pkg/util"
 )
 
-// ScaleSpec describes the attributes a Scale subresource
+// ScaleSpec describes the attributes of a Scale subresource
 type ScaleSpec struct {
-	// Replicas is the number of desired replicas. More info: http://releases.k8s.io/HEAD/docs/user-guide/replication-controller.md#what-is-a-replication-controller"
+	// Replicas is the number of desired replicas. Setting this number will set the number of the desired replicas in the underlying ReplicationController/Deployment.
```

Comment:
For all API field and type comments, do not use Go field and type names. See:
https://github.com/kubernetes/kubernetes/pull/15350#discussion_r41797272
https://github.com/kubernetes/kubernetes/pull/15350#discussion_r41797383
https://github.com/kubernetes/kubernetes/pull/15350#discussion_r41797661